Content.Server/Singularity/EntitySystems/EmitterSystem.cs

```diff
@@ -156,6 +190,16 @@ private void OnActivate(EntityUid uid, EmitterComponent component, ActivateInWor
             }
         }
 
+        private void OnPowerOff(EntityUid uid, EmitterComponent component, PowerOffDoAfterEvent args)
+        {
+            if (args.Cancelled)
+                return;
+
+            SwitchOff(uid, component);
+            _popup.PopupEntity(Loc.GetString("comp-emitter-turned-off",
+                ("target", uid)), uid, args.User);
+        }
```

Comment:
```suggestion
        private void OnPowerOff(EntityUid uid, EmitterComponent component, PowerOffDoAfterEvent args) # Ratbite - Start
        {
            if (args.Cancelled)
                return;

            SwitchOff(uid, component);
            _popup.PopupEntity(Loc.GetString("comp-emitter-turned-off",
                ("target", uid)), uid, args.User);
        } # Ratbite  - End
```
Shoving this much code into upstream systems is a little gross but, alas.